SEP/SEP-0007/sep-0007.md

```diff
@@ -25,11 +25,11 @@ This SEP describes the issues and proposes solutions.
 <p>The SPARQL 1.1 algebra operation
 &ldquo;<a href="https://www.w3.org/TR/sparql11-query/#defn_substitute">substitute</a>&rdquo; evaluates
 a graph pattern where there are specific values for variables given by a solution
-mapping. The operation is used to in the evaluation of <tt>EXISTS</tt> and
```

Comment:
Nothing down this change. In case the Markdown conversion is accepted to redo it in that version.